its/src/test/java/com/sonar/maven/it/suite/DependencyTest.java

```diff
@@ -42,7 +42,7 @@
   @Before
   public void deleteData() {
     // Design features have been dropped in 5.2
-    assumeTrue(!orchestrator.getServer().version().isGreaterThanOrEquals("5.2"));
+    assumeTrue(!orchestrator.getServer().version().isGreaterThanOrEquals(5, 2));
```

Comment:
Can be completely dropped IMO, since we run QA on 6.7+

Reply:
Ok, good point